packages/odata-service-inquirer/src/translations/odata-service-inquirer.i18n.json

```diff
@@ -114,48 +114,48 @@
             "choiceCloudFoundry": "Discover a Cloud Foundry Service"
         },
         "serviceKey": {
-            "message": "Service key file path",
+            "message": "Service Key File Path",
             "hint": "Select a local file that defines the service connection for an ABAP Environment on SAP Business Technology Platform.",
-            "incompleteServiceKeyInfo": "Service keys file does not contain the required information",
-            "unparseableServiceKey": "Service keys file contents are not a valid JSON format"
+            "incompleteServiceKeyInfo": "The service keys file does not contain the required information. Check the file is valid and try again.",
+            "unparseableServiceKey": "The service keys file does not contain a valid JSON format. Check the formatting and try again."
         },
         "cloudFoundryAbapSystem": {
             "message": "ABAP environment",
             "hint": "Enter the name of the Cloud Foundry service that contains the ABAP Environment instance."
         },
         "destinationServicePath": {
-            "message": "Service path",
-            "hint": "Enter the path to the OData service, relative to the selected destination URL.",
+            "message": "Service Path",
+            "hint": "Enter the path to the OData service relative to the selected destination URL.",
             "invalidServicePathWarning": "Please enter a valid service path"
         },
         "mainEntitySelection": {
-            "message": "Main entity",
+            "message": "Main Entity",
             "defaultEntityNameNotFoundWarning": "The supplied entity cannot be found in the service. Please choose from the list above.",
-            "noDraftEnabledEntitiesError": "The CAP service you have chosen does not have any entities that are draft enabled. In order to generate a V4 Fiori application using this floor plan for a CAP project, the entity selected must be draft enabled.",
+            "noDraftEnabledEntitiesError": "The CAP service you have chosen does not have any entities that are draft enabled. To generate a V4 Fiori application using this floor plan for a CAP project, the entity selected must be draft enabled.",
             "noEntitiesError": "$t(errors.noRelevantEntities)",
-            "noEntitiesAlpV4Error": "The OData V4 service you have provided is not suitable for use in an Analytical List Page application. The service must contain aggregate based entities for this template."
+            "noEntitiesAlpV4Error": "The OData V4 service you have provided is not suitable for use in an analytical list page application. The service must contain aggregate based entities for this template."
```

Comment:
Are we sure that floorplan names should not be capitalised?  e.g. "Analytical List Page"